vlib/x/README.md

```diff
@@ -0,0 +1,14 @@
+## Description:
+
+`x` is a module that...
+
+*TODO*
+
+## Examples:
+
+```v
+import x
+
+// TODO: CODE HERE
+
+```
```

Comment:
`x` is not a module by itself, it is just a namespace for the modules inside.
The reason they are under `x/` and not directly under `vlib/` is because they are considered incomplete/experimental.